### DISCUSSION TITLE
[JS101] 關於open index.html

### DISCUSSION THREAD

**Discussion by ming320003** [Lidemy/forum#37] [前端相關問題]
標題：[FE201] 關於open index.html
內文：

### 碰到的問題
關於在瀏覽器裡寫javascript單元。照著做後，並沒有產出和老師一樣的結果。
首先是問題一，
![圖片_2021-01-02_152438](https://user-images.githubusercontent.com/76862990/103452838-9fc06880-4d0e-11eb-8473-6d34af8f45f9.png)
我打開vim後，老師的畫面會出現一條橫線，但是我的沒有。請問這會有影響嗎？

問題二，回到原本的頁面後，輸入open index.html
卻出現了bash: open: command not found

上網查了一下，https://stackoverflow.com/questions/32793782/git-bash-open-command-windows/51813397
根據第一個解答，我將open改為explorer，IE瀏覽器就自己跳出來了。我看了一下console區域，只有IE瀏覽器才有改變，其他瀏覽器沒有。(我通常是使用opera)，我想請問所以windows在這步驟究竟要輸入甚麼呢？
![圖片_2021-01-02_154602](https://user-images.githubusercontent.com/76862990/103453120-9d134280-4d11-11eb-8b4b-cb440a5742d8.png)

謝謝

**Comment by aszx87410** [comment]
> 我打開vim後，老師的畫面會出現一條橫線，但是我的沒有。請問這會有影響嗎？

不會喔，這沒有影響，只是畫面顯示的差別而已，應該是因為我有調一些設定

> 根據第一個解答，我將open改為explorer，IE瀏覽器就自己跳出來了。我看了一下console區域，只有IE瀏覽器才有改變，其他瀏覽器沒有。(我通常是使用opera)，我想請問所以windows在這步驟究竟要輸入甚麼呢？

open 其實就只是用另一個方式幫你開啟 index.html 這檔案而已，你也可以直接點兩下 index.html，一樣可以開啟，或者是右鍵然後用 opera 開啟之類的，結果都會是一樣的，只要能在瀏覽器裡面打開 index.html 這個檔案即可

**Reply by ming320003** [reply]
謝謝老師，已經成功在opera開啟了:)
